packages/patternfly-4/react-core/src/components/EmptyState/EmptyStateIcon.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { css } from '@patternfly/react-styles';
+import styles from '@patternfly/patternfly/components/EmptyState/empty-state.css';
+import { Omit } from '../../helpers/typeUtils';
+
+export enum IconSize {
+  sm = 'sm',
+  md = 'md',
+  lg ='lg',
+  xl ='xl'
+}
+
+export interface IconProps extends Omit<React.HTMLProps<SVGElement>, 'size'> {
+  color?: string;
+  size?: IconSize;
+  title?: string;
+}
+
+export interface EmptyStateIconProps extends IconProps {
+  /** Additional classes added to the EmptyState */
+  className?: string;
+  /** Icon component to be rendered inside the EmptyState */
+  icon: string | React.FunctionComponent<IconProps>;
```

Comment:
I think `string | React.ReactType<IconProps>` might be a more appropriate type if it works.